sycl/include/CL/sycl/detail/scheduler/commands.hpp

```diff
@@ -290,6 +294,25 @@ class ExecCGCommand : public Command {
   std::unique_ptr<detail::CG> MCommandGroup;
 };
 
+class UpdateHostRequirementCommand : public Command {
+public:
+  UpdateHostRequirementCommand(QueueImplPtr Queue, Requirement *Req,
+                               AllocaCommandBase *AllocaForReq)
+      : Command(CommandType::UPDATE_REQUIREMENT, std::move(Queue)),
+        MReqToUpdate(Req), MAllocaForReq(AllocaForReq),
+        MStoredRequirement(*Req) {}
+
+  Requirement *getStoreRequirement() { return &MStoredRequirement; }
```

Comment:
Method is called getStoreRequirement and variable is MStore**d**Requirement. Shouldn't they be aligned?

Reply:
Yes, will fix this typo.